web/src/pages/billing/SequencingGroupsByMonth.tsx

```diff
@@ -101,21 +150,100 @@ const SequencingGroupsByMonthDisplay: React.FunctionComponent<ISGByMonthDisplayP
     const [measureRef, { width, height }] = useMeasure<HTMLDivElement>()
 
+    // Merges counts, grouped by either type or technology.
+    const mergeTypeOrTechCount = React.useCallback(
+        (gqlData: ISequencingGroupQueryData[], groupBy: string) => {
+            if (groupBy != 'type' && groupBy != 'technology') {
+                return []
+            }
+
+            const intermediateData = gqlData.reduce(
+                (acc, item) => {
+                    const dateKey: string = item.date
+                    const groupKey: string = item[groupBy]
+
+                    if (!acc[dateKey]) acc[dateKey] = {}
+                    if (!acc[dateKey][groupKey]) acc[dateKey][groupKey] = 0
+
+                    acc[dateKey][groupKey] += item.count
+
+                    return acc
+                },
+                {} as Record<string, Record<string, number>>
+            )
+
+            // Convert back into an array.
+            const result: ITypeData[] = []
+            Object.entries(intermediateData).forEach(([dateStr, typeMap]) => {
+                const date = new Date(dateStr)
+
+                Object.entries(typeMap).forEach(([groupKey, count]) => {
+                    result.push({ date, type: groupKey, grouping: groupKey, count })
+                })
+            })
+            return result
+        },
+        []
+    )
+
+    // Merges all counts, grouped by date.
+    const mergeAllCounts = React.useCallback((gqlData: ISequencingGroupQueryData[]) => {
+        const intermediateData = gqlData.reduce(
+            (acc, item) => {
+                const dateKey: string = item.date
+
+                if (!acc[dateKey]) acc[dateKey] = 0
+
+                acc[dateKey] += item.count
+                return acc
+            },
+            {} as Record<string, number>
+        )
+
+        // Convert back into an array.
+        const result: ITypeData[] = []
+        Object.entries(intermediateData).forEach(([dateStr, count]) => {
+            const date = new Date(dateStr)
+            result.push({ date, type: 'Sequencing Groups', grouping: 'Sequencing Groups', count })
+        })
+
+        return result
+    }, [])
+
     // Converting the date that GraphQL gives from a string to a Date type.
     const plotData = React.useMemo(() => {
         if (loading || error || !data) return []
 
-        return data.project.sequencingGroupHistory.map((item) => {
-            return {
-                ...item,
-                date: new Date(item.date),
-            } as ITypeData
-        })
-    }, [data, loading, error])
+        // Breakdown by both Type and Technology, which is already how the data is returned from the backend.
+        if (sgBreakdown == SeqGroupBreakdown.Both) {
+            return data.project.sequencingGroupHistory.map((item) => {
+                return {
+                    date: new Date(item.date),
+                    type: item.type + ': ' + item.technology,
```

Comment:
This could be formatted a bit more nicely, maybe `${item.type} / ${item.technology}`?